giraffe/package.json

```diff
@@ -13,6 +13,18 @@
     "prettier:fix": "prettier --config ../.prettierrc.json --write '{src,../stories/src}/**/*.{ts,tsx}'",
     "typecheck": "tsc --noEmit --pretty"
   },
+  "keywords": [
+    "influxdb",
+    "giraffe",
+    "graph",
+    "plot",
+    "visualization",
+    "layer",
+    "line",
+    "scatterplot",
+    "heatmap",
+    "histogram"
```

Comment:
should we consider adding something about `data` as a key word? `data-visualization`?

Reply:
Sure, i will throw in `data`

Reply:
`stat`, `time series`